src/HumusAmqpModule/Consumer.php

```diff
@@ -263,42 +263,40 @@ public function consume($msgAmount = 0)
     {
         $this->target = $msgAmount;
 
-        foreach ($this->queues as $index => $queue) {
-            if (!$this->timestampLastAck) {
-                $this->timestampLastAck = microtime(1);
-            }
-
-            $message = $queue->get();
+        /** @var \AMQPQueue $queue */
+        foreach($this->queues as $queue) {
+            $queue->consume(function ($message, $queue) {
```

Comment:
in v1.0.0 there is only one queue per consumer. But in earlier releases there are multiple queues per consumer allowed. That's why I think the consume-method does not makes sense, unless there is a way to call consume() on multiple queues (without forking or threading).